examples/restful-google-custom-method.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	. "github.com/emicklei/go-restful"
+	"io"
+	"log"
+	"net/http"
+)
+
+// This example shows how to create a Route with google custom method
+// Requires the use of a CurlyRouter and path should end with the custom method
+//
+// GET http://localhost:8080/resource/some-resource-id:init
+
+func main() {
+	DefaultContainer.Router(CurlyRouter{})
+	ws := new(WebService)
+	ws.Route(ws.GET("/basepath/{resourceId}:init").To(fromPathParam))
```

Comment:
i think it is good to add another Route to demonstrate that the verb is causing another function to be called.

so next to :init have :reset or some other verb you can think of.

Reply:
@emicklei , well, i have already fixed example